README.md

```diff
@@ -1,8 +1,8 @@
 # ACE: AI2 Climate Emulator
 This repo contains the inference code accompanying "ACE: A fast, skillful learned global atmospheric model for climate prediction" ([arxiv:2310.02074](https://arxiv.org/abs/2310.02074)).
 
-## DISCLAIMER
-This is rapidly changing research software. We make no guarantees of maintaining backwards compatibility.
+## Documentation
```

Comment:
It was intentional—we have a note in the docs about this being research software which I think is sufficient.

Sounds good, I'll remove the documentation section. I don't think we need to bother with local build instructions since we'll have the RTD page up soon.